productize-the-model.adoc

```diff
@@ -91,13 +91,26 @@ image::che-pylint.png[che-pylint]
 The python code generated from the notebook has been refactored into a
 python class and visualization removed.
 
+
 == Refactor the New Model
 
 Next, we will then modify the code into a format that the pipeline can
 run to train and build the image with the model. The pipeline will call
 `train-stage.sh` and expects the model to be written to a folder at
 `/workspace/model`. 
 
+We have prepared the refactored model at `/projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.ans.py`. Copy it to `/projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.py`
+
+[source,bash,role="copypaste"]
+----
+cp /projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.ans.py /projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.py
+----
+
+Copying the file over would change the contents of the lr.py file 
+
+(From file editor select "Open Files" -> rh-mlops-workshop -> src -> train -> lr.py)
```

Comment:
You are hardcoding the directory name. 

You can provide a screenshot here. You can just use the file explorer.